ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -505,11 +506,15 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
         }
         for item in items {
             if item.tag == BarButtonItemTag.dismiss.rawValue {
-                item.badgeValue = isOn ? "12345" : nil
+                let badgeValue = isOn ? "12345" : nil
+                let badgeAccessibilityLabel = isOn ? "items" : nil
```

Comment:
Shouldn't the badgeAccessibilityLabel be set to "\(badgeValue) items" instead of just "items"?

Reply:
Actually, earlier my design was that we are going to ask user only the suffx string and we will put badge value on our own but now I realized that was not good instead we should be asking the complete badge accessibility label from the user. So yes, now I have fixed.